tests/test_series.py

```diff
@@ -242,8 +246,8 @@ def test_types_drop_multilevel() -> None:
 
 
 def test_types_dropna() -> None:
-    s = pd.Series([1, np.nan, np.nan])
-    check(assert_type(s.dropna(), pd.Series), pd.Series)
+    s = pd.Series([1.0, np.nan])
```

Comment:
pyright would otherwise infer it as `Series[int | float]` while mypy infers `Series[float]`

Reply:
OK.